aezeed/cipherseed_rpctest.go

```diff
@@ -1,5 +1,4 @@
-//go:build rpctest
-// +build rpctest
+//go:build integration
```

Comment:
Should we rename the files that have the `_rpctest` suffix to `_integration` as well?

Reply:
good idea, switched.